index.html

```diff
@@ -310,6 +319,224 @@ <h2>Contact Me</h2>
       
   <hr>
     </section> 
-  
+    <section class="works-desktop">
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_157](https://user-images.githubusercontent.com/69638013/167258031-89cf3cb1-37c1-4ba6-b00a-4c8cd66918fb.png)  |  ![Screenshot_156](https://user-images.githubusercontent.com/69638013/167258048-ead8e41f-372b-4d09-9969-5e3baae6e04b.png)

- [ ] Kindly align the li tags for each card to look more like the Figma design, some of the tags are not in the center of the background and some are aligned to the right

#### OPTIONAL

- [ ] Please add the rounded borders to each card on desktop

- [ ] Kindly align all cards in this section to the center

- [ ] On mobile version please ensure that each li tags in the cards are aligned properly and have the same styling

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_160](https://user-images.githubusercontent.com/69638013/167258404-1cc3e5f8-b692-40e1-b6eb-04943e8c79ac.png)  |  ![Screenshot_159](https://user-images.githubusercontent.com/69638013/167258338-3561b22b-27a0-4c7c-9f8f-84c8bfd7f536.png)